diff --git a/node_modules/strapi/tests/filtering.test.e2e.js b/node_modules/strapi/tests/filtering.test.e2e.js
index 68564ff..625709f 100644
--- a/node_modules/strapi/tests/filtering.test.e2e.js
+++ b/node_modules/strapi/tests/filtering.test.e2e.js
@@ -78,7 +78,7 @@ const productFixtures = [
     price: null,
     decimal_field: 142.43,
     rank: 142,
-    big_rank: 345678912983,
+    big_rank: '345678912983', // Need to compare against string, it gets coerced
   },
 ];
 
diff --git a/node_modules/strapi/tests/search.test.e2e.js b/node_modules/strapi/tests/search.test.e2e.js
index f44c86e..b494ba5 100644
--- a/node_modules/strapi/tests/search.test.e2e.js
+++ b/node_modules/strapi/tests/search.test.e2e.js
@@ -60,7 +60,7 @@ const bedFixtures = [
     sku: 'sleepybed_0152',
     savEmail: 'sav@bed.fr',
     type: 'foam',
-    serialNumber: 999999999999999,
+    serialNumber: '999999999999999', // Need to compare against string, it gets coerced
     peopleNumber: 6,
     fabricThickness: 1.14157,
   },
@@ -73,7 +73,7 @@ const bedFixtures = [
     sku: 'tiredbed_0001',
     savEmail: 'sav@sleep.fr',
     type: 'feather',
-    serialNumber: 1111111111111111,
+    serialNumber: '1111111111111111', // Need to compare against string, it gets coerced
     peopleNumber: 1,
     fabricThickness: 1.0001,
   },
