diff --git a/node_modules/strapi/__tests__/filtering.test.e2e.js b/node_modules/strapi/__tests__/filtering.test.e2e.js
index 84d5980..a19c12b 100644
--- a/node_modules/strapi/__tests__/filtering.test.e2e.js
+++ b/node_modules/strapi/__tests__/filtering.test.e2e.js
@@ -79,7 +79,7 @@ const productFixtures = [
     price: null,
     decimal_field: 142.43,
     rank: 142,
-    big_rank: 345678912983,
+    big_rank: '345678912983', // Need to compare against string, it gets coerced
   },
 ];
 
diff --git a/node_modules/strapi/__tests__/search.test.e2e.js b/node_modules/strapi/__tests__/search.test.e2e.js
index 551a60b..4651a0a 100644
--- a/node_modules/strapi/__tests__/search.test.e2e.js
+++ b/node_modules/strapi/__tests__/search.test.e2e.js
@@ -59,7 +59,7 @@ const bedFixtures = [
     sku: 'sleepybed_0152',
     savEmail: 'sav@bed.fr',
     type: 'foam',
-    serialNumber: 999999999999999,
+    serialNumber: '999999999999999', // Need to compare against string, it gets coerced
     peopleNumber: 6,
     fabricThickness: 1.14157,
   },
@@ -72,7 +72,7 @@ const bedFixtures = [
     sku: 'tiredbed_0001',
     savEmail: 'sav@sleep.fr',
     type: 'feather',
-    serialNumber: 1111111111111111,
+    serialNumber: '1111111111111111', // Need to compare against string, it gets coerced
     peopleNumber: 1,
     fabricThickness: 1.0001,
   },
