diff --git a/node_modules/strapi/lib/Strapi.js b/node_modules/strapi/lib/Strapi.js
index 2b720d3..4a005f1 100644
--- a/node_modules/strapi/lib/Strapi.js
+++ b/node_modules/strapi/lib/Strapi.js
@@ -188,11 +188,11 @@ class Strapi {
       // Is the project initialised?
       const isInitialised = await utils.isInitialised(this);
 
-      if (!isInitialised) {
-        this.logFirstStartupMessage();
-      } else {
-        this.logStartupMessage();
-      }
+      // if (!isInitialised) {
+      //   this.logFirstStartupMessage();
+      // } else {
+      //   this.logStartupMessage();
+      // }
 
       // Emit started event.
       await this.telemetry.send('didStartServer');
@@ -235,7 +235,7 @@ class Strapi {
     }
 
     // Kill process.
-    process.exit(exitCode);
+    //process.exit(exitCode);
   }
 
   async load() {
